packages/core/src/api/webhook.ts

```diff
@@ -59,9 +65,13 @@ export class WebhooksAPI {
 	public async edit(
 		id: Snowflake,
 		webhook: RESTPatchAPIWebhookJSONBody,
-		{ token, reason }: { reason?: string; token?: string } = {},
+		{ token, reason, signal }: { reason?: string | undefined; signal?: AbortSignal | undefined; token?: string } = {},
```

Comment:
We can probably use `Pick<> & { token?: string | undefined }` here, likewise everywhere else in this file.

Otherwise, `token?: string` → `token?: string | undefined`.